Presentation.Web/app/models/data-processing/data-processing-agreementDTO.ts

```diff
@@ -8,6 +8,18 @@
     export interface IDataProcessingAgreementDTO {
         id: number,
         name: string,
+        organizationId: number,
+        references: Array<dpaReference>;
```

Comment:
Typenavne er med PascalCase... Navnet er mærkeligt. Du mener vel DataProcessingReference...

Reply:
Renamer til IDataProcessingReferenceDTO for at matche de ændre.